utils/build/docker/golang/app/gin/main.go

```diff
@@ -27,14 +28,23 @@ func main() {
 	r.Any("/", func(ctx *gin.Context) {
 		ctx.Writer.WriteHeader(http.StatusOK)
 	})
+	r.Any("/stats-unique", func(ctx *gin.Context) {
```

Comment:
Can you document in `docs/weblog` what this endpoint is supposed to do ?

Reply:
This was already documented on `docs/weblog/README.md` (within this PR). Should it be placed somewhere else?

Reply:
indeed, sorry